qutebrowser/mainwindow/statusbar/bar.py

```diff
@@ -104,14 +110,26 @@ class StatusBar(QWidget):
             {{ color['statusbar.bg'] }}
         }
 
-        QWidget#StatusBar[insert_active="true"] {
-            {{ color['statusbar.bg.insert'] }}
+        QWidget#StatusBar QLabel {
+            {{ color['statusbar.fg'] }}
+        }
+
+        QWidget#StatusBar QLineEdit {
+            {{ color['statusbar.fg'] }}
+        }
```

Comment:
Can't you combine those two to say `QWidget#StatusBar QLabel QLineEdit {` (like the deleted one at the bottom of the stylesheet)?